lib/bulk_data_test_kit/v1.0.1/bulk_data_output_check_test.rb

```diff
@@ -24,6 +24,16 @@ class BulkDataOutputCheckTest < Inferno::Test
       DESCRIPTION
       # link 'http://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#response---complete-status'
 
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@33',
+                            'hl7.fhir.uv.bulkdata_2.0.0@34',
+                            'hl7.fhir.uv.bulkdata_2.0.0@35',
+                            'hl7.fhir.uv.bulkdata_2.0.0@36',
```

Comment:
So I seem to have accidentally placed duplicate references to these requirements. 
I have checked each requirement and there shouldn't be any more duplicates. 

I first placed them here on the basis that this is checking the output. I then found what I think is a better place for them and forgot to remove these.

I placed them in [bulk_data_valid_resources_test.rb](https://github.com/inferno-framework/bulk-data-test-kit/blob/37c3a4b447b1a69d9b8fb08a0a3b81460ef35629/lib/bulk_data_test_kit/v1.0.1/bulk_data_valid_resources_test.rb#L38) because perform_bulk_export_validation calls [check_file_request](https://github.com/inferno-framework/bulk-data-test-kit/blob/9e9d8831810168f850e9fd84993be74c7a8ca8d8/lib/bulk_data_test_kit/bulk_export_validation_tester.rb#L74) and in that function it appears  that it is checking resourceType against what is defined in the request, [which should be set to CompartmentDefinition](https://www.hl7.org/fhir/compartmentdefinition-patient.json.html) for patient compartment. 

so it relies on the resourceType defined in the request, and if it is compartment patient it should be checking for that. 

This portion of the codebase has a lot going on so if i interpreted that incorrectly I can remove the references to 33-36 from bulk_data_valid_resources_test.rb.